src/lib/DAL/RoomMethods.js

```diff
@@ -1,13 +1,19 @@
 // /* eslint-disable */
-/* global Rooms:true RoomMethods:true ProgramUtils:true RandomGenerate:true */
+/* global Rooms:true RoomMethods:true KeyNotes:true
+Keynotes:true ProgramUtils:true RandomGenerate:true */
 
 Rooms = new Mongo.Collection("rooms");
+KeyNotes = new Mongo.Collection("keynotes");
 if(Meteor.isServer){
     Meteor.publish("rooms", function(){
         return Rooms.find({});
     });
+    Meteor.publish("keynotes", function(){
+        return KeyNotes.find({});
```

Comment:
I think we should only subscribe to keynotes to the room the user is in, not whole table.